.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
Why though?

Reply:
Because we don't want anybody to commit the phpstorm project config?

Reply:
Ah, I missed that. Did not see that the file was .gitignore. Assumed that it was a committed dir . . . oh well  . . Assumption is the mother of all f*ckups :-)